src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+
+    }
```

Comment:
Please remove the redundant empty lines after the method signature (line 5) and before the closing brace of the method (line 12). According to the checklist, we don't need them after class declaration or method signature.